src/localizer.cpp

```diff
@@ -260,6 +263,17 @@ QString Localizer::localizedCityName(const QString& code, const QString& city) {
   return ServerI18N::translateCityName(code, city);
 }
 
+QLocale Localizer::locale() {
+  logger.debug() << "Set locale";
+
+  if (!loadLanguageInternal(SettingsHolder::instance()->languageCode())) {
```

Comment:
You don't need to do this. You can simply return `m_locale`. No QML code can run before the loading of the languages.

Reply:
Basically, remove this method :)